extensions/job_management/standard/recommendations/job-management/create/REC_body-openeo.adoc

```diff
@@ -0,0 +1,8 @@
+[[rec_job-management_create_body-openeo]]
+[recommendation]
+====
+[%metadata]
+label:: /rec/job-management/create/body-openeo
+
+part:: If the job can be encoded as an <<rc_openeo,OpenEO graph>>, implementation SHOULD consider supporting the <<rc_openeo,OpenEO graph>> encoding.
```

Comment:
The question for openEO is, what you actually submit here? A job or a process? I guess it's a job description - and then I'm wondering whether there's a way to align that a bit more across encodings/specs...

Reply:
For submission, it would be the contents of https://api.openeo.org/#tag/Batch-Jobs/operation/create-job
For the response, the https://api.openeo.org/#tag/Batch-Jobs/operation/describe-job

I'd expect `Content-Schema: https://openeo.org/documentation/1.0/developers/api/openapi.yaml#/components/schemas/batch_job`, but let us know if there is a more appropriate reference.